panel/io/reload.py

```diff
@@ -133,20 +141,35 @@ async def setup_autoreload_watcher(stop_event=None):
 def watch(filename):
     """
     Add a file to the watch list.
-
-    All imported modules are watched by default.
     """
     _watched_files.add(filename)
 
+def is_subpath(subpath, path):
+    try:
+        return bool(pathlib.Path(subpath).relative_to(pathlib.Path(path)))
```

Comment:
```suggestion
        return os.path.commonpath([path, subpath]) == path
```

Reply:
This could also be: `os.path.abspath(subpath).startswith(os.path.abspath(path))`